internal/provider/resource_auth0_branding.go

```diff
@@ -151,17 +160,28 @@ func updateBranding(ctx context.Context, d *schema.ResourceData, m interface{})
 func deleteBranding(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
 	api := m.(*management.Management)
 
-	tenant, err := api.Tenant.Read()
-	if err != nil {
-		return diag.FromErr(err)
+	if _, ok := d.GetOk("universal_login"); !ok {
+		d.SetId("")
+		return nil
 	}
 
-	if tenant.Flags.GetEnableCustomDomainInEmails() {
-		if err = api.Branding.DeleteUniversalLogin(); err != nil {
-			return diag.FromErr(err)
+	if err := checkForCustomDomains(api); err != nil {
+		d.SetId("")
+		return diag.Diagnostics{
+			{
+				Severity: diag.Warning,
```

Comment:
Decided to set this to a warning instead of an error when trying to destroy the auth0_branding resource.